src/ENDFtk/syntaxTree/File/test/File.test.cpp

```diff
@@ -29,10 +29,16 @@ SCENARIO( "Creating a syntax tree of an ENDF File" ){
           < syntaxTree::File< std::string::iterator > >
           ( head, start, beg, end, lin );
         
+        THEN( "We can exstract the MT numbers" ){
+          std::vector< int > refMTs{ 1, 2, 102 };
+          CHECK( refMTs == original->sectionNumbers() );
```

Comment:
related to previous comment

```suggestion
          CHECK( ranges::equal( refMTs,original->sectionNumbers() ) );
```